src/core/reducers/categories.js

```diff
@@ -31,6 +31,14 @@ type CategoryMapType = {
   },
 };
 
+export type CategorySlugType = {
```

Comment:
This doesn't appear to be used anywhere.

Reply:
Yes, I am testing some type variable with this, but just forgot to remove. I will update this.